src/modes/install.ts

```diff
@@ -39,7 +46,30 @@ export default async function(pkgs: PackageRequirement[]) {
         if (program.includes("{{")) continue
 
         const pkgstr = utils.pkg.str(pkg)
-        const exec = `exec pkgx +${pkgstr} -- ${program} "$@"`
+        if (UNSAFE) {
+          const parts = pkgstr.split("/")
+          parts.pop()
+          await Deno.mkdir(Path.home().join(`.cache/pkgx/envs/${parts.join("/")}`).toString(), {recursive: true})
```

Comment:
this should almost certainly be
```typescript
  config.cache.join(`pkgx/envs/${parts.join("/"}`).mkdir('p')
```
to use our standard `Path` library and `Config` struct.

Reply:
Is that `'p'` argument the option? (as in `mkdir -p`)

Reply:
yep: https://github.com/pkgxdev/libpkgx/blob/2aec73a0c16222d2c1a4eddb96acc867dddaa191/src/utils/Path.ts#L351-L357

Reply:
k

Reply:
From where that `config` var comes though? (Sorry for being an idiot, lol)

Reply:
Is that from `useConfig()`?

Reply:
Not at all. it should be from `const config = useConfig()` where `Config` comes from https://github.com/pkgxdev/libpkgx/blob/main/src/hooks/useConfig.ts

Reply:
That's what I meant:
```ts
const config = useConfig();
config.cache.join(`envs/${parts.join("/")}/`).mkdir("p");
```

Reply:
lol, `not at all` was in re: your idiot remark. we crossed each other.

Reply:
xD